books/views.py

```diff
@@ -53,7 +53,7 @@
 from opds import generate_tags_catalog
 from opds import generate_taggroups_catalog
 
-from pathagar.books.app_settings import BOOK_PUBLISHED
+from pathagar.books.app_settings import BOOK_PUBLISHED, ALLOW_USER_COMMENTS
```

Comment:
This should use:
`from django.conf import settings` and `settings.ALLOW_USER_COMMENTS`

`django.conf.settings` allows users to extend the base level settings file with a `local_settings.py` on deployment that may or may not overwrite the `app_settings` version.  This is the [recommended way for accessing django settings](https://docs.djangoproject.com/en/1.8/topics/settings/#using-settings-in-python-code).

I'll open an issue to refactor this in the rest of the app.